metricbeat/module/kubernetes/state_deployment/_meta/test/ksm.v2.6.0.plain.expected

```diff
@@ -59,6 +59,10 @@
 		"MetricSetFields": {
 			"name": "coredns",
 			"paused": false,
+			"status": {
+				"available": "true",
+				"progressing": "true"
+        	},
```

Comment:
This bracket should be one layer in, right? Like https://github.com/elastic/beats/pull/35999/files#diff-050ad6dbacd48af0a47953cade64df8f81c2104c232ddc15dfc1feff29d48badR40-R43.


Same for the rest of them.

Reply:
Blew my brain :) Fixed thanks @ChrsMark